test/mitmproxy/console/test_flowlist.py

```diff
@@ -4,16 +4,7 @@
 from mitmproxy import options
 from .. import tservers
 import pytest
-from unittest import mock
-
-
-class UrlError(Exception):
-    pass
-
-
-def mock_log(message):
-    if "Invalid URL" in message:
-        raise UrlError(message)
+from unittest import mock as Mock
```

Comment:
You don't have to rename the import - my comment from before was just an example.